api/api.yaml

```diff
@@ -2477,6 +2477,78 @@ paths:
             application/problem+json:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
+  /orgs/{orgId}/billingAccount:
+    get:
+      summary: Get the billing account for an organization
+      operationId: getOrgBillingAccount
+      tags: [ "Billing Account" ]
+      parameters:
+        - name: orgId
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Billing account
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BillingAccount"
+        404:
+          description: Not Found
+          content:
+            application/problem+json:
+              schema:
+                $ref: "../problem/problem.yaml#/components/schemas/InputValidationProblem"
+        default:
+          description: Error
+          content:
+            application/problem+json:
+              schema:
+                $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
+  /billingAccounts/{billingAccountId}/session:
```

Comment:
even though it's more verbose, can you change `session` to `portalSession`? Otherwise it's not intuitive what a "session for a billing account" means.